post/inventory_model.php

```diff
@@ -0,0 +1 @@
+<?php
```

Comment:
?

Reply:
This was to remind me to implement the post in a model. Good catch!